packages/firebase_remote_config/firebase_remote_config/example/lib/main.dart

```diff
@@ -1,97 +1,33 @@
-// ignore_for_file: require_trailing_commas
 // Copyright 2019 The Chromium Authors. All rights reserved.
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-import 'dart:async';
-
 import 'package:firebase_core/firebase_core.dart';
-import 'package:firebase_remote_config/firebase_remote_config.dart';
+import 'package:firebase_remote_config_example/home_page.dart';
 import 'package:flutter/material.dart';
-import 'package:flutter/services.dart';
 
 import 'firebase_options.dart';
 
-void main() {
+void main() async {
   WidgetsFlutterBinding.ensureInitialized();
-  runApp(MaterialApp(
-      title: 'Remote Config Example',
-      home: FutureBuilder<FirebaseRemoteConfig>(
-        future: setupRemoteConfig(),
-        builder: (BuildContext context,
-            AsyncSnapshot<FirebaseRemoteConfig> snapshot) {
-          return snapshot.hasData
-              ? WelcomeWidget(remoteConfig: snapshot.requireData)
-              : Container();
-        },
-      )));
+  await Firebase.initializeApp(
```

Comment:
I'm happy you removed the builder setup, it's been something I've wanted to get around to doing!

Reply:
Other than possibly adding `UnsupportedError `, LGTM.